src/convert.js

```diff
@@ -98,9 +98,7 @@ function convertLine(ring, out, tolerance, isPolygon) {
         const x = projectX(ring[j][0]);
         const y = projectY(ring[j][1]);
 
-        out.push(x);
-        out.push(y);
-        out.push(0);
+        out.push(x, y, 0);
```

Comment:
I generally avoid multi-argument `push` because it consistently showed slowed performance than monomorphic calls. V8 might optimize this now (not sure) but I'd be on the safe side.

Reply:
Just tested and the multi-args is marginally faster, see https://jsperf.com/push-single-vs-multi-args

Also it is used at https://github.com/mapbox/geojson-vt/blob/master/src/index.js#L147 so it would be better for consistency.

Thanks !